
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://CHENyiru3.github.io/Yiru_study_in_zje/IBMS3_full/Yiru/detailed/6.%20Critical%20thinking/">
      
      
      
      
        
      
      
      <link rel="icon" href="../../../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.1">
    
    
      
        <title>6. Critical thinking - ZJE Collection</title>
      
    
    
      <link rel="stylesheet" href="../../../../assets/stylesheets/main.484c7ddc.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../../../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#dna" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../../.." title="ZJE Collection" class="md-header__button md-logo" aria-label="ZJE Collection" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            ZJE Collection
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              6. Critical thinking
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../../.." title="ZJE Collection" class="md-nav__button md-logo" aria-label="ZJE Collection" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    ZJE Collection
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Home
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Courses
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            
  
    Courses
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../BG2_sum/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BG2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../GP2_sum/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    GP2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IBMS3
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../MBE3_sum/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MBE3
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../PoN3_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    PoN3
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../IID_4_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IID_4
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../IFBS2_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IFBS2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../IN_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IN
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../BaO2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BaO2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../MI2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MI2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Contributors
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Contributors
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../AUTHORS/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Authors
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../ZIPS_INDEX/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Zips Index
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../CONTRIBUTING/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Contributing
  

    
  </span>
  
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#dna" class="md-nav__link">
    <span class="md-ellipsis">
      
        DNA评估方法
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="DNA评估方法">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-pcrqpcr" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. PCR/qPCR
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-southernnorthern-blotting" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. Southern/Northern Blotting
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rna" class="md-nav__link">
    <span class="md-ellipsis">
      
        RNA评估方法
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="RNA评估方法">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-rt-pcr" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. RT-PCR
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-rna-seq" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. RNA-seq
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#-dnarna" class="md-nav__link">
    <span class="md-ellipsis">
      
        蛋白质-DNA/RNA相互作用
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="蛋白质-DNA/RNA相互作用">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-chip-seqcutrun" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. ChIP-seq/CUT&amp;RUN
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-emsa" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. EMSA
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        蛋白质分析方法
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="蛋白质分析方法">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-western-blot" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. Western Blot
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-co-ipms" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. Co-IP/MS
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#_2" class="md-nav__link">
    <span class="md-ellipsis">
      
        表观遗传学技术
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="表观遗传学技术">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-bisulfite-sequencing" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. Bisulfite Sequencing
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-atac-seq" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. ATAC-seq
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#_3" class="md-nav__link">
    <span class="md-ellipsis">
      
        基因编辑与筛选
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="基因编辑与筛选">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-crispr-cas9" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. CRISPR-Cas9
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-crispr-screen" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. CRISPR Screen
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#_4" class="md-nav__link">
    <span class="md-ellipsis">
      
        计算生物学方法
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="计算生物学方法">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-gwaseqtl" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. GWAS/eQTL
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. 深度学习模型
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#_5" class="md-nav__link">
    <span class="md-ellipsis">
      
        通用改进策略
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#_6" class="md-nav__link">
    <span class="md-ellipsis">
      
        关键总结
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#dna-assessment-methods" class="md-nav__link">
    <span class="md-ellipsis">
      
        DNA Assessment Methods
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="DNA Assessment Methods">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-pcrqpcr_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. PCR/qPCR
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-southern-blotting" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. Southern Blotting
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#rna-assessment-methods" class="md-nav__link">
    <span class="md-ellipsis">
      
        RNA Assessment Methods
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="RNA Assessment Methods">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-rt-pcr_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. RT-PCR
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-rna-seq_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. RNA-seq
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protein-dnarna-interaction-methods" class="md-nav__link">
    <span class="md-ellipsis">
      
        Protein-DNA/RNA Interaction Methods
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Protein-DNA/RNA Interaction Methods">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-chip-seqcutrun_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. ChIP-seq/CUT&amp;RUN
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-emsa_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. EMSA
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protein-analysis-methods" class="md-nav__link">
    <span class="md-ellipsis">
      
        Protein Analysis Methods
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Protein Analysis Methods">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-western-blot_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. Western Blot
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-co-ipms_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. Co-IP/MS
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#epigenetic-sequencing" class="md-nav__link">
    <span class="md-ellipsis">
      
        Epigenetic Sequencing
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Epigenetic Sequencing">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-bisulfite-sequencing_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. Bisulfite Sequencing
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-atac-seq_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. ATAC-seq
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gene-editing-screening" class="md-nav__link">
    <span class="md-ellipsis">
      
        Gene Editing &amp; Screening
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Gene Editing &amp; Screening">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-crispr-cas9_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. CRISPR-Cas9
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-crispr-screen_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. CRISPR Screen
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#computational-methods" class="md-nav__link">
    <span class="md-ellipsis">
      
        Computational Methods
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Computational Methods">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#1-gwaseqtl_1" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. GWAS/eQTL
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#2-deep-learning-models" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. Deep Learning Models
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#universal-improvement-strategies" class="md-nav__link">
    <span class="md-ellipsis">
      
        Universal Improvement Strategies
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#key-takeaways" class="md-nav__link">
    <span class="md-ellipsis">
      
        Key Takeaways
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#critical-thinking-sections-for-molecular-biology-experiments" class="md-nav__link">
    <span class="md-ellipsis">
      
        Critical Thinking Sections for Molecular Biology Experiments
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Critical Thinking Sections for Molecular Biology Experiments">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#dna-assessment-techniques" class="md-nav__link">
    <span class="md-ellipsis">
      
        DNA Assessment Techniques:
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-assessment-techniques" class="md-nav__link">
    <span class="md-ellipsis">
      
        RNA Assessment Techniques:
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#protein-dna-interaction-techniques" class="md-nav__link">
    <span class="md-ellipsis">
      
        Protein-DNA Interaction Techniques:
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#protein-analysis-techniques" class="md-nav__link">
    <span class="md-ellipsis">
      
        Protein Analysis Techniques:
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gene-expression-sequencing" class="md-nav__link">
    <span class="md-ellipsis">
      
        Gene Expression Sequencing:
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#epigenetics-sequencing" class="md-nav__link">
    <span class="md-ellipsis">
      
        Epigenetics Sequencing:
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gene-editing" class="md-nav__link">
    <span class="md-ellipsis">
      
        Gene Editing:
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-critical-analysis-similar-to-question-1" class="md-nav__link">
    <span class="md-ellipsis">
      
        Example Critical Analysis (Similar to Question 1):
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  



<p>以下是针对您列出的分子生物学实验技术和计算方法的Critical Thinking分析，包括潜在偏差(Bias)和改进方向(Improvements)：</p>
<hr />
<h3 id="dna"><strong>DNA评估方法</strong><a class="headerlink" href="#dna" title="Permanent link">&para;</a></h3>
<h4 id="1-pcrqpcr">1. <strong>PCR/qPCR</strong><a class="headerlink" href="#1-pcrqpcr" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>引物二聚体和非特异性扩增导致假阳性</li>
<li>扩增效率差异影响定量准确性（qPCR）</li>
<li>对抑制剂敏感（如血液中的肝素）</li>
<li><strong>Improvements</strong>: </li>
<li>使用数字PCR(dPCR)提高绝对定量精度</li>
<li>加入内参基因(如GAPDH)校正</li>
</ul>
<h4 id="2-southernnorthern-blotting">2. <strong>Southern/Northern Blotting</strong><a class="headerlink" href="#2-southernnorthern-blotting" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>低通量，一次只能检测少量目标</li>
<li>需要大量起始材料（尤其Northern对RNA降解敏感）</li>
<li>杂交探针可能与非目标序列交叉反应</li>
<li><strong>Improvements</strong>: </li>
<li>结合毛细管电泳提高分辨率</li>
<li>用荧光标记替代放射性标记</li>
</ul>
<hr />
<h3 id="rna"><strong>RNA评估方法</strong><a class="headerlink" href="#rna" title="Permanent link">&para;</a></h3>
<h4 id="1-rt-pcr">1. <strong>RT-PCR</strong><a class="headerlink" href="#1-rt-pcr" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>反转录效率受RNA二级结构影响</li>
<li>引物设计偏向已知序列，难检测新异构体</li>
<li><strong>Improvements</strong>: </li>
<li>使用随机六聚体+oligo(dT)混合引物</li>
<li>加入外源RNA spike-in控制</li>
</ul>
<h4 id="2-rna-seq">2. <strong>RNA-seq</strong><a class="headerlink" href="#2-rna-seq" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>3'端偏好性（尤其单细胞RNA-seq）</li>
<li>GC含量偏差影响测序深度</li>
<li>rRNA去除不彻底</li>
<li><strong>Improvements</strong>: </li>
<li>使用UMI消除PCR重复</li>
<li>长读长测序(PacBio)解决剪接异构体问题</li>
</ul>
<hr />
<h3 id="-dnarna"><strong>蛋白质-DNA/RNA相互作用</strong><a class="headerlink" href="#-dnarna" title="Permanent link">&para;</a></h3>
<h4 id="1-chip-seqcutrun">1. <strong>ChIP-seq/CUT&amp;RUN</strong><a class="headerlink" href="#1-chip-seqcutrun" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>抗体特异性问题（假阳性结合）</li>
<li>交联效率影响信号强度</li>
<li>对弱结合位点灵敏度低</li>
<li><strong>Improvements</strong>: </li>
<li>使用多克隆抗体混合提高覆盖度</li>
<li>CUT&amp;Tag替代减少细胞用量</li>
</ul>
<h4 id="2-emsa">2. <strong>EMSA</strong><a class="headerlink" href="#2-emsa" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>仅体外验证，无法反映细胞内环境</li>
<li>无法区分直接/间接结合</li>
<li><strong>Improvements</strong>: </li>
<li>结合超迁移(supershift)验证特定蛋白</li>
</ul>
<hr />
<h3 id="_1"><strong>蛋白质分析方法</strong><a class="headerlink" href="#_1" title="Permanent link">&para;</a></h3>
<h4 id="1-western-blot">1. <strong>Western Blot</strong><a class="headerlink" href="#1-western-blot" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>抗体交叉反应性</li>
<li>线性动态范围窄（约10^2）</li>
<li><strong>Improvements</strong>: </li>
<li>用质谱验证抗体特异性</li>
<li>近红外荧光检测提高灵敏度</li>
</ul>
<h4 id="2-co-ipms">2. <strong>Co-IP/MS</strong><a class="headerlink" href="#2-co-ipms" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>无法区分直接/间接互作</li>
<li>高丰度蛋白掩盖低丰度信号</li>
<li><strong>Improvements</strong>: </li>
<li>交联质谱(XL-MS)捕获瞬态互作</li>
<li>亲和纯化后定量(SILAC)</li>
</ul>
<hr />
<h3 id="_2"><strong>表观遗传学技术</strong><a class="headerlink" href="#_2" title="Permanent link">&para;</a></h3>
<h4 id="1-bisulfite-sequencing">1. <strong>Bisulfite Sequencing</strong><a class="headerlink" href="#1-bisulfite-sequencing" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>DNA降解（亚硫酸氢盐处理）</li>
<li>无法区分5mC/5hmC</li>
<li><strong>Improvements</strong>: </li>
<li>OxBS-seq特异性检测5hmC</li>
<li>单细胞甲基化测序</li>
</ul>
<h4 id="2-atac-seq">2. <strong>ATAC-seq</strong><a class="headerlink" href="#2-atac-seq" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>对核小体定位分辨率有限(~200bp)</li>
<li>转座酶偏好性(GC-rich区域)</li>
<li><strong>Improvements</strong>: </li>
<li>结合MNase-seq提高核小体定位精度</li>
</ul>
<hr />
<h3 id="_3"><strong>基因编辑与筛选</strong><a class="headerlink" href="#_3" title="Permanent link">&para;</a></h3>
<h4 id="1-crispr-cas9">1. <strong>CRISPR-Cas9</strong><a class="headerlink" href="#1-crispr-cas9" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>脱靶效应（尤其低效sgRNA）</li>
<li>非同源末端连接(NHEJ)引入不可控indel</li>
<li><strong>Improvements</strong>: </li>
<li>使用高保真Cas9变体(eSpCas9)</li>
<li>单碱基编辑避免DSB</li>
</ul>
<h4 id="2-crispr-screen">2. <strong>CRISPR Screen</strong><a class="headerlink" href="#2-crispr-screen" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>文库覆盖不均导致假阴性</li>
<li>多基因协同效应被忽略</li>
<li><strong>Improvements</strong>: </li>
<li>组合筛选(Combi-Seq)</li>
<li>动态追踪单细胞谱系</li>
</ul>
<hr />
<h3 id="_4"><strong>计算生物学方法</strong><a class="headerlink" href="#_4" title="Permanent link">&para;</a></h3>
<h4 id="1-gwaseqtl">1. <strong>GWAS/eQTL</strong><a class="headerlink" href="#1-gwaseqtl" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>人群分层导致假阳性</li>
<li>无法确定因果变异</li>
<li><strong>Improvements</strong>: </li>
<li>孟德尔随机化验证因果关系</li>
<li>跨种族meta分析</li>
</ul>
<h4 id="2">2. <strong>深度学习模型</strong><a class="headerlink" href="#2" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>: </li>
<li>对生物可解释性差（黑箱问题）</li>
<li>依赖训练数据质量</li>
<li><strong>Improvements</strong>: </li>
<li>注意力机制可视化关键特征</li>
<li>迁移学习解决小样本问题</li>
</ul>
<hr />
<h3 id="_5"><strong>通用改进策略</strong><a class="headerlink" href="#_5" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>实验设计</strong>：</li>
<li>增加生物学重复(n≥3)</li>
<li>使用正交方法验证（如WB验证RNA-seq）</li>
<li><strong>数据分析</strong>：</li>
<li>严格多重检验校正（FDR&lt;0.05）</li>
<li>公开代码和原始数据(FAIR原则)</li>
<li><strong>技术整合</strong>：</li>
<li>多组学联合分析（如ATAC+RNA-seq）</li>
<li>活细胞成像动态验证</li>
</ol>
<hr />
<h3 id="_6"><strong>关键总结</strong><a class="headerlink" href="#_6" title="Permanent link">&para;</a></h3>
<ul>
<li><strong>灵敏度 vs 特异性</strong>：几乎所有技术都面临此权衡（如CRISPR的脱靶）</li>
<li><strong>还原论局限</strong>：体外实验无法完全模拟体内环境（建议类器官模型）</li>
<li><strong>技术迭代</strong>：长读长测序、空间组学等新技术正在解决传统偏差</li>
</ul>
<p>如需对某一技术深入讨论，可提供具体方向。</p>
<hr />
<p>Here is the organized critical analysis of each listed methodology in English, including potential biases and improvement strategies:</p>
<hr />
<h3 id="dna-assessment-methods"><strong>DNA Assessment Methods</strong><a class="headerlink" href="#dna-assessment-methods" title="Permanent link">&para;</a></h3>
<h4 id="1-pcrqpcr_1">1. <strong>PCR/qPCR</strong><a class="headerlink" href="#1-pcrqpcr_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Primer dimer formation and nonspecific amplification (false positives).  </li>
<li>Variable amplification efficiency affecting quantification accuracy (qPCR).  </li>
<li>Sensitivity to inhibitors (e.g., heparin in blood samples).  </li>
<li><strong>Improvements</strong>:  </li>
<li>Use digital PCR (dPCR) for absolute quantification.  </li>
<li>Include internal reference genes (e.g., GAPDH) for normalization.  </li>
</ul>
<h4 id="2-southern-blotting">2. <strong>Southern Blotting</strong><a class="headerlink" href="#2-southern-blotting" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Low throughput (limited to single-target detection).  </li>
<li>High RNA degradation risk (Northern blotting).  </li>
<li>Cross-hybridization with non-target sequences.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Combine with capillary electrophoresis for higher resolution.  </li>
<li>Replace radioactive probes with fluorescence-based detection.  </li>
</ul>
<hr />
<h3 id="rna-assessment-methods"><strong>RNA Assessment Methods</strong><a class="headerlink" href="#rna-assessment-methods" title="Permanent link">&para;</a></h3>
<h4 id="1-rt-pcr_1">1. <strong>RT-PCR</strong><a class="headerlink" href="#1-rt-pcr_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Reverse transcription efficiency affected by RNA secondary structures.  </li>
<li>Primer bias against novel isoforms/unknown sequences.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Use random hexamer + oligo(dT) hybrid primers.  </li>
<li>Add exogenous RNA spike-ins for quality control.  </li>
</ul>
<h4 id="2-rna-seq_1">2. <strong>RNA-seq</strong><a class="headerlink" href="#2-rna-seq_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>3'-end bias (especially in single-cell RNA-seq).  </li>
<li>GC content bias influencing sequencing depth.  </li>
<li>Incomplete rRNA depletion.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Implement unique molecular identifiers (UMIs) to remove PCR duplicates.  </li>
<li>Use long-read sequencing (PacBio) to resolve splice isoforms.  </li>
</ul>
<hr />
<h3 id="protein-dnarna-interaction-methods"><strong>Protein-DNA/RNA Interaction Methods</strong><a class="headerlink" href="#protein-dnarna-interaction-methods" title="Permanent link">&para;</a></h3>
<h4 id="1-chip-seqcutrun_1">1. <strong>ChIP-seq/CUT&amp;RUN</strong><a class="headerlink" href="#1-chip-seqcutrun_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Antibody specificity issues (false-positive binding).  </li>
<li>Crosslinking efficiency impacting signal strength.  </li>
<li>Low sensitivity for weak binding sites.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Combine multiple antibodies to improve coverage.  </li>
<li>Adopt CUT&amp;Tag for lower input requirements.  </li>
</ul>
<h4 id="2-emsa_1">2. <strong>EMSA</strong><a class="headerlink" href="#2-emsa_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Limited to <em>in vitro</em> conditions (no cellular context).  </li>
<li>Unable to distinguish direct vs. indirect binding.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Combine with supershift assays to confirm protein identity.  </li>
</ul>
<hr />
<h3 id="protein-analysis-methods"><strong>Protein Analysis Methods</strong><a class="headerlink" href="#protein-analysis-methods" title="Permanent link">&para;</a></h3>
<h4 id="1-western-blot_1">1. <strong>Western Blot</strong><a class="headerlink" href="#1-western-blot_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Antibody cross-reactivity.  </li>
<li>Narrow linear dynamic range (~10²-fold).  </li>
<li><strong>Improvements</strong>:  </li>
<li>Validate antibodies with mass spectrometry.  </li>
<li>Use near-infrared fluorescence for enhanced sensitivity.  </li>
</ul>
<h4 id="2-co-ipms_1">2. <strong>Co-IP/MS</strong><a class="headerlink" href="#2-co-ipms_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Inability to differentiate direct vs. indirect interactions.  </li>
<li>Signal masking by high-abundance proteins.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Apply crosslinking mass spectrometry (XL-MS) to capture transient interactions.  </li>
<li>Use SILAC labeling for quantitative analysis.  </li>
</ul>
<hr />
<h3 id="epigenetic-sequencing"><strong>Epigenetic Sequencing</strong><a class="headerlink" href="#epigenetic-sequencing" title="Permanent link">&para;</a></h3>
<h4 id="1-bisulfite-sequencing_1">1. <strong>Bisulfite Sequencing</strong><a class="headerlink" href="#1-bisulfite-sequencing_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>DNA degradation during bisulfite treatment.  </li>
<li>Indistinguishable 5mC/5hmC signals.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Use oxidative bisulfite sequencing (OxBS-seq) for 5hmC detection.  </li>
<li>Implement single-cell methylation sequencing.  </li>
</ul>
<h4 id="2-atac-seq_1">2. <strong>ATAC-seq</strong><a class="headerlink" href="#2-atac-seq_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Limited nucleosome positioning resolution (~200 bp).  </li>
<li>Transposase preference for GC-rich regions.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Integrate MNase-seq for precise nucleosome mapping.  </li>
</ul>
<hr />
<h3 id="gene-editing-screening"><strong>Gene Editing &amp; Screening</strong><a class="headerlink" href="#gene-editing-screening" title="Permanent link">&para;</a></h3>
<h4 id="1-crispr-cas9_1">1. <strong>CRISPR-Cas9</strong><a class="headerlink" href="#1-crispr-cas9_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Off-target effects (especially with low-efficiency sgRNAs).  </li>
<li>Uncontrolled indels via NHEJ repair.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Use high-fidelity Cas9 variants (e.g., eSpCas9).  </li>
<li>Apply base editing to avoid double-strand breaks.  </li>
</ul>
<h4 id="2-crispr-screen_1">2. <strong>CRISPR Screen</strong><a class="headerlink" href="#2-crispr-screen_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Library coverage bias leading to false negatives.  </li>
<li>Oversimplification of gene synergy effects.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Perform combinatorial CRISPR screening (Combi-Seq).  </li>
<li>Track single-cell lineages dynamically.  </li>
</ul>
<hr />
<h3 id="computational-methods"><strong>Computational Methods</strong><a class="headerlink" href="#computational-methods" title="Permanent link">&para;</a></h3>
<h4 id="1-gwaseqtl_1">1. <strong>GWAS/eQTL</strong><a class="headerlink" href="#1-gwaseqtl_1" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Population stratification causing false positives.  </li>
<li>Difficulty in identifying causal variants.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Apply Mendelian randomization for causal inference.  </li>
<li>Conduct cross-ancestry meta-analyses.  </li>
</ul>
<h4 id="2-deep-learning-models">2. <strong>Deep Learning Models</strong><a class="headerlink" href="#2-deep-learning-models" title="Permanent link">&para;</a></h4>
<ul>
<li><strong>Bias</strong>:  </li>
<li>Poor biological interpretability ("black box" issue).  </li>
<li>Dependency on training data quality.  </li>
<li><strong>Improvements</strong>:  </li>
<li>Visualize key features via attention mechanisms.  </li>
<li>Use transfer learning for small datasets.  </li>
</ul>
<hr />
<h3 id="universal-improvement-strategies"><strong>Universal Improvement Strategies</strong><a class="headerlink" href="#universal-improvement-strategies" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>Experimental Design</strong>:  </li>
<li>Include biological replicates (n ≥ 3).  </li>
<li>Validate with orthogonal methods (e.g., WB for RNA-seq results).  </li>
<li><strong>Data Analysis</strong>:  </li>
<li>Apply strict multiple testing correction (FDR &lt; 0.05).  </li>
<li>Follow FAIR principles (Findable, Accessible, Interoperable, Reusable).  </li>
<li><strong>Technology Integration</strong>:  </li>
<li>Multi-omics integration (e.g., ATAC-seq + RNA-seq).  </li>
<li>Live-cell imaging for dynamic validation.  </li>
</ol>
<hr />
<h3 id="key-takeaways"><strong>Key Takeaways</strong><a class="headerlink" href="#key-takeaways" title="Permanent link">&para;</a></h3>
<ul>
<li><strong>Sensitivity-Specificity Tradeoff</strong>: Inherent to most technologies (e.g., CRISPR off-targets).  </li>
<li><strong>Reductionism Limitations</strong>: <em>In vitro</em> models cannot fully replicate <em>in vivo</em> complexity (consider organoid models).  </li>
<li><strong>Emerging Solutions</strong>: Long-read sequencing, spatial omics, and single-cell technologies address traditional biases.  </li>
</ul>
<h1 id="critical-thinking-sections-for-molecular-biology-experiments">Critical Thinking Sections for Molecular Biology Experiments<a class="headerlink" href="#critical-thinking-sections-for-molecular-biology-experiments" title="Permanent link">&para;</a></h1>
<h2 id="dna-assessment-techniques">DNA Assessment Techniques:<a class="headerlink" href="#dna-assessment-techniques" title="Permanent link">&para;</a></h2>
<ol>
<li><strong>PCR</strong></li>
<li><strong>Limitations</strong>: Cannot quantify initial DNA amount, prone to contamination, limited size range of amplicons, potential primer-dimer formation.</li>
<li><strong>Controls Needed</strong>: Positive control (known template), negative control (no template), internal control for inhibition.</li>
<li>
<p><strong>Alternative Approaches</strong>: Digital PCR for absolute quantification, long-range PCR for larger fragments.</p>
</li>
<li>
<p><strong>qPCR</strong></p>
</li>
<li><strong>Limitations</strong>: Requires careful primer design, affected by PCR inhibitors, amplification efficiency variations.</li>
<li><strong>Data Analysis Considerations</strong>: Proper threshold setting, appropriate reference genes, efficiency corrections for accurate quantification.</li>
<li>
<p><strong>Validation Requirements</strong>: Standard curves, melt curve analysis, no-template controls.</p>
</li>
<li>
<p><strong>Southern Blotting</strong></p>
</li>
<li><strong>Limitations</strong>: Low throughput, labor-intensive, requires large DNA amounts, limited sensitivity.</li>
<li><strong>Critical Parameters</strong>: Probe specificity, transfer efficiency, blocking effectiveness.</li>
<li><strong>Modern Alternatives</strong>: qPCR, digital PCR, or NGS approaches that offer higher sensitivity and throughput.</li>
</ol>
<h2 id="rna-assessment-techniques">RNA Assessment Techniques:<a class="headerlink" href="#rna-assessment-techniques" title="Permanent link">&para;</a></h2>
<ol>
<li><strong>RT-PCR</strong></li>
<li><strong>Limitations</strong>: Reverse transcriptase variability, RNA degradation risks, genomic DNA contamination.</li>
<li><strong>Controls Required</strong>: No-RT controls, reference gene normalization, RNA quality assessment.</li>
<li>
<p><strong>Optimization Factors</strong>: RNA preservation methods, DNase treatment, RT enzyme selection.</p>
</li>
<li>
<p><strong>Northern Blotting</strong></p>
</li>
<li><strong>Limitations</strong>: Low sensitivity, time-consuming, requires substantial RNA input.</li>
<li><strong>Critical Considerations</strong>: RNA integrity, efficient transfer, probe specificity.</li>
<li><strong>Modern Alternatives</strong>: RNA-seq, NanoString, or RT-qPCR for higher sensitivity and throughput.</li>
</ol>
<h2 id="protein-dna-interaction-techniques">Protein-DNA Interaction Techniques:<a class="headerlink" href="#protein-dna-interaction-techniques" title="Permanent link">&para;</a></h2>
<ol>
<li><strong>ChIP-seq</strong></li>
<li><strong>Limitations</strong>: Antibody specificity issues, high background, formaldehyde crosslinking biases.</li>
<li><strong>Controls Required</strong>: Input DNA, IgG controls, spike-in normalization.</li>
<li>
<p><strong>Validation Approaches</strong>: Replicate concordance, motif enrichment analysis, orthogonal techniques like CUT&amp;RUN.</p>
</li>
<li>
<p><strong>Footprinting Assay</strong></p>
</li>
<li><strong>Limitations</strong>: Limited to in vitro interactions, requires optimization for each protein-DNA pair.</li>
<li><strong>Critical Parameters</strong>: Nuclease concentration, incubation time, protein concentration.</li>
<li>
<p><strong>Alternative Methods</strong>: In vivo techniques like ChIP-seq or CUT&amp;RUN for physiological context.</p>
</li>
<li>
<p><strong>EMSA</strong></p>
</li>
<li><strong>Limitations</strong>: Semi-quantitative, artificial binding conditions, limited to small DNA fragments.</li>
<li><strong>Controls Needed</strong>: Unlabeled competitor DNA, non-specific competitor, supershift controls.</li>
<li>
<p><strong>Optimization Factors</strong>: Binding buffer composition, protein:DNA ratio, gel percentage.</p>
</li>
<li>
<p><strong>CUT&amp;RUN</strong></p>
</li>
<li><strong>Limitations</strong>: Antibody specificity dependencies, optimization required for each target.</li>
<li><strong>Critical Considerations</strong>: Nuclease concentration, antibody selection, cell permeabilization.</li>
<li><strong>Advantages Over ChIP</strong>: Lower background, reduced input requirements, higher resolution.</li>
</ol>
<h2 id="protein-analysis-techniques">Protein Analysis Techniques:<a class="headerlink" href="#protein-analysis-techniques" title="Permanent link">&para;</a></h2>
<ol>
<li><strong>Western Blotting</strong></li>
<li><strong>Limitations</strong>: Semi-quantitative, antibody cross-reactivity, limited dynamic range.</li>
<li><strong>Controls Required</strong>: Loading controls, molecular weight markers, positive/negative controls.</li>
<li>
<p><strong>Optimization Factors</strong>: Transfer conditions, blocking reagents, antibody dilutions.</p>
</li>
<li>
<p><strong>Co-IP</strong></p>
</li>
<li><strong>Limitations</strong>: Non-physiological buffer conditions, transient interactions may be missed.</li>
<li><strong>Controls Needed</strong>: IgG control, input samples, reciprocal IP validation.</li>
<li>
<p><strong>Critical Parameters</strong>: Lysis conditions, antibody specificity, washing stringency.</p>
</li>
<li>
<p><strong>Pull-down Assay</strong></p>
</li>
<li><strong>Limitations</strong>: Artificial conditions, tag interference with interactions, non-specific binding.</li>
<li><strong>Controls Required</strong>: Tag-only controls, competitive inhibition controls.</li>
<li><strong>Validation Approaches</strong>: Reciprocal pull-downs, dose-dependent competition, orthogonal methods.</li>
</ol>
<h2 id="gene-expression-sequencing">Gene Expression Sequencing:<a class="headerlink" href="#gene-expression-sequencing" title="Permanent link">&para;</a></h2>
<ol>
<li><strong>RNA-seq</strong></li>
<li><strong>Limitations</strong>: 3' bias in some protocols, GC content biases, batch effects.</li>
<li><strong>Quality Controls</strong>: RNA integrity assessment, spike-in controls, technical replicates.</li>
<li>
<p><strong>Analysis Considerations</strong>: Appropriate normalization methods, batch correction, differential expression statistics.</p>
</li>
<li>
<p><strong>Single-cell RNA-seq</strong></p>
</li>
<li><strong>Limitations</strong>: Dropout events, cell capture biases, amplification biases.</li>
<li><strong>Critical Parameters</strong>: Cell viability, doublet rate, sequencing depth.</li>
<li><strong>Analytical Challenges</strong>: Dimensionality reduction choices, clustering parameters, trajectory inference assumptions.</li>
</ol>
<h2 id="epigenetics-sequencing">Epigenetics Sequencing:<a class="headerlink" href="#epigenetics-sequencing" title="Permanent link">&para;</a></h2>
<ol>
<li><strong>ATAC-seq</strong></li>
<li><strong>Limitations</strong>: Open chromatin doesn't always indicate functional activity, cell type heterogeneity effects.</li>
<li><strong>Controls Required</strong>: Naked DNA controls, mitochondrial DNA exclusion, technical replicates.</li>
<li>
<p><strong>Validation Approaches</strong>: Correlation with DNase-seq, functional validation of identified regions (as in the example question).</p>
</li>
<li>
<p><strong>ChIP-seq for Histone Modifications</strong></p>
</li>
<li><strong>Limitations</strong>: Antibody specificity concerns, crosslinking biases, batch effects.</li>
<li><strong>Controls Needed</strong>: Input normalization, spike-in controls, validation with different antibodies.</li>
<li><strong>Interpretation Challenges</strong>: Correlative vs. causative relationships, combinatorial modification effects.</li>
</ol>
<h2 id="gene-editing">Gene Editing:<a class="headerlink" href="#gene-editing" title="Permanent link">&para;</a></h2>
<ol>
<li><strong>CRISPR-Cas9</strong></li>
<li><strong>Limitations</strong>: Off-target effects, PAM site requirements, delivery efficiency.</li>
<li><strong>Controls Required</strong>: Non-targeting gRNAs, wild-type cells, validation of edits.</li>
<li>
<p><strong>Optimization Factors</strong>: gRNA design, delivery method, timing of analysis post-editing.</p>
</li>
<li>
<p><strong>RNAi</strong></p>
</li>
<li><strong>Limitations</strong>: Off-target effects, incomplete knockdown, transient effects.</li>
<li><strong>Controls Needed</strong>: Scrambled siRNA controls, multiple siRNAs per target, rescue experiments.</li>
<li><strong>Validation Requirements</strong>: Protein-level knockdown confirmation, phenotype specificity tests.</li>
</ol>
<h2 id="example-critical-analysis-similar-to-question-1">Example Critical Analysis (Similar to Question 1):<a class="headerlink" href="#example-critical-analysis-similar-to-question-1" title="Permanent link">&para;</a></h2>
<p>For investigating if an ATAC-seq identified open chromatin region is an enhancer for gene X:</p>
<ol>
<li>
<p><strong>Experimental Method</strong>: CRISPR interference (CRISPRi) using dCas9-KRAB to selectively repress the potential enhancer region, followed by gene X expression analysis.</p>
</li>
<li>
<p><strong>Alternative Approaches</strong>: </p>
</li>
<li>CRISPR deletion of the region followed by expression analysis</li>
<li>Reporter assays with the region cloned upstream of a minimal promoter</li>
<li>
<p>Chromosome conformation capture (3C/4C) to detect physical interactions with the gene promoter</p>
</li>
<li>
<p><strong>Controls Required</strong>:</p>
</li>
<li>Cells with non-targeting gRNA</li>
<li>Targeting a known non-regulatory region</li>
<li>
<p>Targeting the promoter as a positive control</p>
</li>
<li>
<p><strong>Potential Confounding Factors</strong>:</p>
</li>
<li>The region might regulate other nearby genes instead of/in addition to gene X</li>
<li>Compensatory mechanisms might mask enhancer function</li>
<li>
<p>Cell type-specific enhancer activity might be missed</p>
</li>
<li>
<p><strong>Validation Strategy</strong>:</p>
</li>
<li>Test multiple independent gRNAs targeting different parts of the region</li>
<li>Perform rescue experiments by re-introducing the enhancer</li>
<li>
<p>Test enhancer activity in multiple cell types relevant to gene X function</p>
</li>
<li>
<p><strong>Quantification and Statistics</strong>:</p>
</li>
<li>Normalize gene X expression to multiple reference genes</li>
<li>Use appropriate statistical tests (t-test or ANOVA) with correction for multiple comparisons</li>
<li>Calculate effect size to determine biological significance beyond statistical significance</li>
</ol>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "../../../..", "features": [], "search": "../../../../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../../../assets/javascripts/bundle.79ae519e.min.js"></script>
      
    
  </body>
</html>