
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://CHENyiru3.github.io/Yiru_study_in_zje/IBMS3_full/Yiru/detailed/experiments_case/MBE%20DNA%20assessment/">
      
      
      
      
        
      
      
      <link rel="icon" href="../../../../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.1">
    
    
      
        <title>MBE DNA assessment - ZJE Collection</title>
      
    
    
      <link rel="stylesheet" href="../../../../../assets/stylesheets/main.484c7ddc.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../../../../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#1-experimental-purpose" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../../../.." title="ZJE Collection" class="md-header__button md-logo" aria-label="ZJE Collection" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            ZJE Collection
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              MBE DNA assessment
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../../../.." title="ZJE Collection" class="md-nav__button md-logo" aria-label="ZJE Collection" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    ZJE Collection
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Home
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Courses
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            
  
    Courses
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../BG2_sum/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BG2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../GP2_sum/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    GP2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IBMS3
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../MBE3_sum/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MBE3
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../PoN3_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    PoN3
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../IID_4_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IID_4
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../IFBS2_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IFBS2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../IN_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IN
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../BaO2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BaO2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../MI2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MI2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Contributors
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Contributors
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../AUTHORS/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Authors
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../ZIPS_INDEX/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Zips Index
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../../../CONTRIBUTING/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Contributing
  

    
  </span>
  
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#1-experimental-purpose" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. Experimental Purpose
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#2-model-system" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. Model System
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#3-measurement-approach" class="md-nav__link">
    <span class="md-ellipsis">
      
        3. Measurement Approach
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#4-group-setting" class="md-nav__link">
    <span class="md-ellipsis">
      
        4. Group Setting
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#5-data-analysis-presentation" class="md-nav__link">
    <span class="md-ellipsis">
      
        5. Data Analysis &amp; Presentation
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#6-technique-comparison" class="md-nav__link">
    <span class="md-ellipsis">
      
        6. Technique Comparison
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#7-complementary-usage-strategy" class="md-nav__link">
    <span class="md-ellipsis">
      
        7. Complementary Usage Strategy
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#complete-experimental-protocols-qpcr-and-southern-blot" class="md-nav__link">
    <span class="md-ellipsis">
      
        Complete Experimental Protocols: qPCR and Southern Blot
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Complete Experimental Protocols: qPCR and Southern Blot">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#qpcr-quantitative-polymerase-chain-reaction-protocol" class="md-nav__link">
    <span class="md-ellipsis">
      
        qPCR (Quantitative Polymerase Chain Reaction) Protocol
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="qPCR (Quantitative Polymerase Chain Reaction) Protocol">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#i-sample-preparation" class="md-nav__link">
    <span class="md-ellipsis">
      
        I. Sample Preparation
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ii-primer-design-and-validation" class="md-nav__link">
    <span class="md-ellipsis">
      
        II. Primer Design and Validation
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iii-qpcr-setup-and-run" class="md-nav__link">
    <span class="md-ellipsis">
      
        III. qPCR Setup and Run
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iv-data-analysis" class="md-nav__link">
    <span class="md-ellipsis">
      
        IV. Data Analysis
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#southern-blot-protocol" class="md-nav__link">
    <span class="md-ellipsis">
      
        Southern Blot Protocol
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Southern Blot Protocol">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#i-dna-extraction-and-preparation" class="md-nav__link">
    <span class="md-ellipsis">
      
        I. DNA Extraction and Preparation
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#ii-gel-electrophoresis-and-transfer" class="md-nav__link">
    <span class="md-ellipsis">
      
        II. Gel Electrophoresis and Transfer
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iii-probe-preparation-and-hybridization" class="md-nav__link">
    <span class="md-ellipsis">
      
        III. Probe Preparation and Hybridization
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#iv-washing-and-detection" class="md-nav__link">
    <span class="md-ellipsis">
      
        IV. Washing and Detection
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#v-membrane-stripping-and-reprobing-optional" class="md-nav__link">
    <span class="md-ellipsis">
      
        V. Membrane Stripping and Reprobing (Optional)
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  



<h2 id="1-experimental-purpose">1. Experimental Purpose<a class="headerlink" href="#1-experimental-purpose" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Scientific Question</strong>: Is gene X present, altered, or differentially expressed in condition Y compared to controls?</li>
<li><strong>Design Rationale</strong>: DNA detection and quantification techniques allow identification of specific sequences, structural variations, and abundance changes</li>
<li><strong>Follow-up Studies</strong>: Functional validation of detected variants, expression analysis, correlation with phenotypic outcomes</li>
</ul>
<h2 id="2-model-system">2. Model System<a class="headerlink" href="#2-model-system" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Primary System</strong>: Human tissue samples from patients with condition Y and matched controls</li>
<li><strong>Rationale</strong>: Patient-derived samples provide direct clinical relevance while allowing sufficient DNA extraction for multiple analytical approaches</li>
<li><strong>Alternatives</strong>:</li>
<li>Cell lines (pros: abundant material, controlled conditions; cons: may not reflect in vivo context)</li>
<li>Animal models (pros: controlled genetics; cons: species differences)</li>
<li>Synthetic DNA (pros: precise control; cons: lacks biological context)</li>
<li><strong>Ethical Considerations</strong>: IRB approval, informed consent, sample de-identification, responsible data sharing</li>
</ul>
<h2 id="3-measurement-approach">3. Measurement Approach<a class="headerlink" href="#3-measurement-approach" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Common Elements</strong>:</li>
<li>High-quality DNA extraction</li>
<li>Appropriate sample storage</li>
<li>Consistent quantification methods</li>
<li>Inclusion of reference standards</li>
<li><strong>Technical Replicates</strong>: Triplicate measurements for each sample and technique</li>
<li><strong>Potential Biases</strong>:</li>
<li>Sample quality variations (standardize extraction protocols)</li>
<li>Batch effects (include inter-run calibrators)</li>
<li>Amplification bias (optimize primer design)</li>
<li>Probe specificity (validate with known controls)</li>
</ul>
<h2 id="4-group-setting">4. Group Setting<a class="headerlink" href="#4-group-setting" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Experimental Groups</strong>:</li>
<li>Test: Samples from subjects with condition Y</li>
<li>Control 1: Matched samples from healthy subjects</li>
<li>Control 2: Positive controls with known sequence variants</li>
<li>Control 3: Negative controls (no template controls)</li>
<li><strong>Controlled Variables</strong>: DNA quantity, quality metrics, reagent lots, instrument calibration</li>
<li><strong>Biological Replicates</strong>: Minimum 20-30 subjects per group for adequate statistical power</li>
<li><strong>Modified Design</strong>: Include family members for hereditary conditions or longitudinal sampling for progressive conditions</li>
</ul>
<h2 id="5-data-analysis-presentation">5. Data Analysis &amp; Presentation<a class="headerlink" href="#5-data-analysis-presentation" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Common Analysis Elements</strong>: </li>
<li>Quality control metrics</li>
<li>Normalization to reference genes/sequences</li>
<li>Statistical comparison between groups</li>
<li>Correlation with clinical parameters</li>
<li><strong>Presentation Approaches</strong>:</li>
<li>Gel/blot images with size markers</li>
<li>Amplification curves and threshold cycles</li>
<li>Quantitative comparisons with error bars</li>
<li>Correlation plots with clinical outcomes</li>
</ul>
<h2 id="6-technique-comparison">6. Technique Comparison<a class="headerlink" href="#6-technique-comparison" title="Permanent link">&para;</a></h2>
<table>
<thead>
<tr>
<th>Feature</th>
<th>PCR</th>
<th>qPCR</th>
<th>Southern Blotting</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Primary Use</strong></td>
<td>Qualitative detection of specific sequences</td>
<td>Precise quantification of target DNA</td>
<td>Detection of specific sequences and structural variations</td>
</tr>
<tr>
<td><strong>Sensitivity</strong></td>
<td>Moderate (detects ~10-100 copies)</td>
<td>High (detects 1-10 copies)</td>
<td>Moderate (50-100 ng of genomic DNA)</td>
</tr>
<tr>
<td><strong>Specificity</strong></td>
<td>Good, primer-dependent</td>
<td>Very good, primer and probe-dependent</td>
<td>Excellent, especially for complex rearrangements</td>
</tr>
<tr>
<td><strong>Quantification</strong></td>
<td>Semi-quantitative at best</td>
<td>Highly quantitative</td>
<td>Semi-quantitative</td>
</tr>
<tr>
<td><strong>Throughput</strong></td>
<td>High</td>
<td>High</td>
<td>Low</td>
</tr>
<tr>
<td><strong>Time Required</strong></td>
<td>2-3 hours</td>
<td>2-3 hours</td>
<td>1-3 days</td>
</tr>
<tr>
<td><strong>Cost</strong></td>
<td>Low</td>
<td>Moderate</td>
<td>High</td>
</tr>
<tr>
<td><strong>Equipment</strong></td>
<td>Basic thermal cycler</td>
<td>qPCR instrument</td>
<td>Multiple specialized equipment</td>
</tr>
<tr>
<td><strong>Technical Expertise</strong></td>
<td>Basic</td>
<td>Moderate</td>
<td>Advanced</td>
</tr>
<tr>
<td><strong>Best For</strong></td>
<td>• Rapid presence/absence detection<br>• Genotyping<br>• Initial screening<br>• Amplifying targets for sequencing</td>
<td>• Precise copy number quantification<br>• Gene expression studies<br>• Pathogen load determination<br>• Allelic discrimination</td>
<td>• Complex structural variations<br>• Large insertions/deletions<br>• Repetitive sequence analysis<br>• Confirming ambiguous PCR results</td>
</tr>
<tr>
<td><strong>Limitations</strong></td>
<td>• Limited quantification<br>• Potential for false positives<br>• Size constraints</td>
<td>• Limited fragment size information<br>• Requires careful assay design<br>• Potential amplification bias</td>
<td>• Labor intensive<br>• Low throughput<br>• Requires large DNA amounts<br>• Radioactive hazards (for some probes)</td>
</tr>
</tbody>
</table>
<h2 id="7-complementary-usage-strategy">7. Complementary Usage Strategy<a class="headerlink" href="#7-complementary-usage-strategy" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Initial Screening</strong>: Use standard PCR for rapid, cost-effective detection of target sequences</li>
<li><strong>Precise Quantification</strong>: Follow with qPCR for accurate measurement of differences between groups</li>
<li><strong>Structural Validation</strong>: Employ Southern blotting to confirm complex structural variations or repetitive sequence changes</li>
<li><strong>Integrated Approach</strong>: Use all three methods for comprehensive characterization of important targets:</li>
<li>PCR to rapidly screen multiple samples</li>
<li>qPCR to precisely quantify differences</li>
<li>Southern blotting to validate structural context and complex variations</li>
</ul>
<p>This multi-technique approach provides complementary data that overcomes the limitations of any single method while maximizing confidence in results for critical findings.</p>
<hr />
<h1 id="complete-experimental-protocols-qpcr-and-southern-blot">Complete Experimental Protocols: qPCR and Southern Blot<a class="headerlink" href="#complete-experimental-protocols-qpcr-and-southern-blot" title="Permanent link">&para;</a></h1>
<h2 id="qpcr-quantitative-polymerase-chain-reaction-protocol">qPCR (Quantitative Polymerase Chain Reaction) Protocol<a class="headerlink" href="#qpcr-quantitative-polymerase-chain-reaction-protocol" title="Permanent link">&para;</a></h2>
<h3 id="i-sample-preparation">I. Sample Preparation<a class="headerlink" href="#i-sample-preparation" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>RNA Extraction</strong></li>
<li>Harvest cells or tissue samples (50-100 mg tissue or 1-5×10^6 cells)</li>
<li>Add 1 mL TRIzol reagent and homogenize</li>
<li>Incubate at room temperature for 5 minutes</li>
<li>Add 0.2 mL chloroform, shake vigorously for 15 seconds</li>
<li>Incubate at room temperature for 3 minutes</li>
<li>Centrifuge at 12,000×g for 15 minutes at 4°C</li>
<li>Transfer aqueous phase to a new tube</li>
<li>Add 0.5 mL isopropanol, incubate for 10 minutes</li>
<li>Centrifuge at 12,000×g for 10 minutes at 4°C</li>
<li>Discard supernatant and wash RNA pellet with 75% ethanol</li>
<li>
<p>Air-dry pellet and resuspend in RNase-free water</p>
</li>
<li>
<p><strong>DNA Removal and RNA Quality Assessment</strong></p>
</li>
<li>Treat RNA with DNase I (1 unit/μg RNA) for 30 minutes at 37°C</li>
<li>Inactivate DNase at 65°C for 10 minutes with EDTA</li>
<li>Measure RNA concentration using spectrophotometer (NanoDrop)</li>
<li>Verify RNA quality by checking A260/A280 ratio (1.8-2.0 is optimal)</li>
<li>
<p>Run 1 μg RNA on a 1% agarose gel to check integrity (28S and 18S rRNA bands)</p>
</li>
<li>
<p><strong>cDNA Synthesis</strong></p>
</li>
<li>Prepare reaction mixture:<ul>
<li>1 μg total RNA</li>
<li>1 μL oligo(dT) primer (50 μM) or random hexamers (50 ng/μL)</li>
<li>1 μL dNTP mix (10 mM each)</li>
<li>Nuclease-free water to 13 μL</li>
</ul>
</li>
<li>Heat at 65°C for 5 minutes, then chill on ice</li>
<li>Add:<ul>
<li>4 μL 5× First-Strand Buffer</li>
<li>1 μL DTT (0.1 M)</li>
<li>1 μL RNase inhibitor (40 units/μL)</li>
<li>1 μL reverse transcriptase (200 units/μL)</li>
</ul>
</li>
<li>Incubate at 42°C for 50 minutes</li>
<li>Inactivate at 70°C for 15 minutes</li>
<li>Store cDNA at -20°C</li>
</ol>
<h3 id="ii-primer-design-and-validation">II. Primer Design and Validation<a class="headerlink" href="#ii-primer-design-and-validation" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>Primer Design</strong></li>
<li>Design primers using software (e.g., Primer3, NCBI Primer-BLAST)</li>
<li>Parameters:<ul>
<li>Amplicon length: 70-150 bp</li>
<li>Primer length: 18-25 nucleotides</li>
<li>GC content: 40-60%</li>
<li>Melting temperature (Tm): 58-62°C</li>
<li>Avoid secondary structures and primer-dimers</li>
</ul>
</li>
<li>Check primer specificity using BLAST</li>
<li>
<p>Order primers from commercial supplier</p>
</li>
<li>
<p><strong>Primer Validation</strong></p>
</li>
<li>Prepare standard PCR reaction:<ul>
<li>12.5 μL 2× PCR master mix</li>
<li>1 μL forward primer (10 μM)</li>
<li>1 μL reverse primer (10 μM)</li>
<li>1 μL cDNA template</li>
<li>9.5 μL nuclease-free water</li>
</ul>
</li>
<li>Run PCR: 95°C for 3 min; 35 cycles of (95°C for 30 sec, 60°C for 30 sec, 72°C for 30 sec); 72°C for 5 min</li>
<li>
<p>Run PCR product on 2% agarose gel to verify single band of expected size</p>
</li>
<li>
<p><strong>Primer Efficiency Test</strong></p>
</li>
<li>Prepare a 5-fold serial dilution of cDNA (1:1, 1:5, 1:25, 1:125, 1:625)</li>
<li>Run qPCR for each dilution in triplicate</li>
<li>Plot Ct values against log(dilution factor)</li>
<li>Calculate efficiency: E = 10^(-1/slope) - 1 (optimal: 90-110%)</li>
</ol>
<h3 id="iii-qpcr-setup-and-run">III. qPCR Setup and Run<a class="headerlink" href="#iii-qpcr-setup-and-run" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>Reaction Preparation</strong></li>
<li>For SYBR Green-based detection:<ul>
<li>10 μL 2× SYBR Green master mix</li>
<li>0.6 μL forward primer (10 μM)</li>
<li>0.6 μL reverse primer (10 μM)</li>
<li>1 μL cDNA template</li>
<li>7.8 μL nuclease-free water</li>
</ul>
</li>
<li>For probe-based detection (TaqMan):<ul>
<li>10 μL 2× TaqMan master mix</li>
<li>1 μL gene-specific primer-probe mix (20×)</li>
<li>1 μL cDNA template</li>
<li>8 μL nuclease-free water</li>
</ul>
</li>
<li>Prepare master mix for all reactions plus 10% extra</li>
<li>Dispense 19 μL master mix into each well of a 96-well plate</li>
<li>Add 1 μL cDNA to appropriate wells</li>
<li>Seal plate with optical adhesive film</li>
<li>
<p>Centrifuge plate briefly to eliminate air bubbles</p>
</li>
<li>
<p><strong>Controls Setup</strong></p>
</li>
<li>No template control (NTC): replace cDNA with water</li>
<li>No reverse transcriptase control (NRT): cDNA synthesis without RT</li>
<li>Positive control: sample known to express target gene</li>
<li>
<p>Reference gene controls: primers for housekeeping genes (GAPDH, β-actin, 18S rRNA)</p>
</li>
<li>
<p><strong>qPCR Cycling Conditions</strong></p>
</li>
<li>For SYBR Green:<ul>
<li>Initial denaturation: 95°C for 10 minutes</li>
<li>40 cycles of:</li>
<li>Denaturation: 95°C for 15 seconds</li>
<li>Annealing/extension: 60°C for 1 minute</li>
<li>Melt curve analysis: 60-95°C (0.3°C increments)</li>
</ul>
</li>
<li>For TaqMan:<ul>
<li>Initial denaturation: 95°C for 10 minutes</li>
<li>40 cycles of:</li>
<li>Denaturation: 95°C for 15 seconds</li>
<li>Annealing/extension: 60°C for 1 minute</li>
</ul>
</li>
</ol>
<h3 id="iv-data-analysis">IV. Data Analysis<a class="headerlink" href="#iv-data-analysis" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>Quality Control</strong></li>
<li>Check amplification plots for smooth, sigmoidal curves</li>
<li>Verify single peak in melt curve analysis (SYBR Green only)</li>
<li>Confirm NTC and NRT controls show no amplification</li>
<li>
<p>Check replicate consistency (Ct standard deviation &lt;0.3)</p>
</li>
<li>
<p><strong>Relative Quantification</strong></p>
</li>
<li>Determine Ct values for target and reference genes</li>
<li>Calculate ΔCt = Ct(target) - Ct(reference)</li>
<li>For comparing experimental vs. control:<ul>
<li>Calculate ΔΔCt = ΔCt(experimental) - ΔCt(control)</li>
<li>Calculate fold change = 2^(-ΔΔCt)</li>
</ul>
</li>
<li>
<p>Apply statistical analysis (t-test or ANOVA)</p>
</li>
<li>
<p><strong>Result Interpretation and Reporting</strong></p>
</li>
<li>Create bar graphs showing relative expression levels</li>
<li>Include error bars (standard deviation or standard error)</li>
<li>Indicate statistical significance (p-values)</li>
<li>Report primer sequences, efficiency, and reference genes used</li>
</ol>
<h2 id="southern-blot-protocol">Southern Blot Protocol<a class="headerlink" href="#southern-blot-protocol" title="Permanent link">&para;</a></h2>
<h3 id="i-dna-extraction-and-preparation">I. DNA Extraction and Preparation<a class="headerlink" href="#i-dna-extraction-and-preparation" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>Genomic DNA Extraction</strong></li>
<li>Harvest cells or tissue samples (100-200 mg tissue or 5-10×10^6 cells)</li>
<li>Add 500 μL lysis buffer (10 mM Tris-HCl pH 8.0, 100 mM NaCl, 25 mM EDTA, 0.5% SDS)</li>
<li>Add proteinase K to 100 μg/mL final concentration</li>
<li>Incubate at 55°C overnight with gentle shaking</li>
<li>Add equal volume of phenol:chloroform:isoamyl alcohol (25:24:1)</li>
<li>Mix thoroughly and centrifuge at 12,000×g for 10 minutes</li>
<li>Transfer aqueous phase to a new tube</li>
<li>Add 0.1 volume of 3M sodium acetate (pH 5.2) and 2 volumes of 100% ethanol</li>
<li>Incubate at -20°C for 30 minutes</li>
<li>Centrifuge at 12,000×g for 15 minutes at 4°C</li>
<li>
<p>Wash pellet with 70% ethanol, air-dry, and resuspend in TE buffer</p>
</li>
<li>
<p><strong>DNA Quality Assessment</strong></p>
</li>
<li>Measure DNA concentration using spectrophotometer (NanoDrop)</li>
<li>Check A260/A280 ratio (1.8-2.0 is optimal)</li>
<li>
<p>Run 1 μg DNA on a 0.8% agarose gel to check integrity (high molecular weight band)</p>
</li>
<li>
<p><strong>Restriction Enzyme Digestion</strong></p>
</li>
<li>Prepare digestion reaction:<ul>
<li>10-20 μg genomic DNA</li>
<li>5 μL 10× restriction enzyme buffer</li>
<li>3-5 units restriction enzyme per μg DNA</li>
<li>Nuclease-free water to 50 μL</li>
</ul>
</li>
<li>Incubate at appropriate temperature (usually 37°C) for 4-16 hours</li>
<li>Add 1/10 volume 3M sodium acetate and 2.5 volumes ethanol</li>
<li>Incubate at -20°C for 30 minutes</li>
<li>Centrifuge at 12,000×g for 15 minutes at 4°C</li>
<li>Wash pellet with 70% ethanol, air-dry, and resuspend in TE buffer</li>
</ol>
<h3 id="ii-gel-electrophoresis-and-transfer">II. Gel Electrophoresis and Transfer<a class="headerlink" href="#ii-gel-electrophoresis-and-transfer" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>Agarose Gel Preparation</strong></li>
<li>Prepare 0.7-0.8% agarose gel in 1× TAE buffer</li>
<li>Add ethidium bromide (0.5 μg/mL) or other DNA stain</li>
<li>
<p>Pour gel and allow to solidify</p>
</li>
<li>
<p><strong>Sample Loading and Electrophoresis</strong></p>
</li>
<li>Mix DNA samples with 6× loading dye</li>
<li>Load samples alongside DNA molecular weight marker</li>
<li>Run gel at 20-30V overnight (12-16 hours) for good separation</li>
<li>Document gel with UV transilluminator</li>
<li>
<p>Trim excess gel and mark orientation</p>
</li>
<li>
<p><strong>Gel Processing</strong></p>
</li>
<li>Depurination: soak gel in 0.25M HCl for 10 minutes with gentle shaking</li>
<li>Rinse briefly with distilled water</li>
<li>Denaturation: soak gel in denaturation solution (0.5M NaOH, 1.5M NaCl) for 30 minutes</li>
<li>Rinse briefly with distilled water</li>
<li>
<p>Neutralization: soak gel in neutralization solution (0.5M Tris-HCl pH 7.5, 1.5M NaCl) for 30 minutes</p>
</li>
<li>
<p><strong>Capillary Transfer Setup</strong></p>
</li>
<li>Cut nylon membrane to gel size and pre-wet in 10× SSC</li>
<li>Place a glass plate across a container filled with 10× SSC</li>
<li>Place a wick (Whatman 3MM paper) over the glass plate with ends in buffer</li>
<li>Place gel upside down on the wick</li>
<li>Place membrane on top of gel (avoid bubbles)</li>
<li>Place 3 sheets of Whatman paper on membrane</li>
<li>Stack paper towels (5-8 cm high) on top</li>
<li>Place weight (~500g) on top</li>
<li>
<p>Allow transfer to proceed overnight (12-16 hours)</p>
</li>
<li>
<p><strong>Post-Transfer Processing</strong></p>
</li>
<li>Disassemble transfer setup and mark membrane orientation</li>
<li>Rinse membrane briefly in 2× SSC</li>
<li>Crosslink DNA to membrane using UV crosslinker (120 mJ/cm²) or bake at 80°C for 2 hours</li>
</ol>
<h3 id="iii-probe-preparation-and-hybridization">III. Probe Preparation and Hybridization<a class="headerlink" href="#iii-probe-preparation-and-hybridization" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>Probe Synthesis</strong></li>
<li>
<p>For radioactive labeling (P32):</p>
<ul>
<li>Prepare reaction mixture:</li>
<li>25-50 ng DNA template</li>
<li>5 μL random primers</li>
<li>Nuclease-free water to 34 μL</li>
<li>Denature at 95°C for 5 minutes, chill on ice</li>
<li>Add:</li>
<li>5 μL 10× buffer</li>
<li>5 μL dNTP mix (minus dCTP)</li>
<li>5 μL [α-32P]dCTP (50 μCi)</li>
<li>1 μL Klenow fragment (5 units)</li>
<li>Incubate at 37°C for 1 hour</li>
<li>Purify labeled probe using spin column</li>
</ul>
</li>
<li>
<p>For non-radioactive labeling (DIG):</p>
<ul>
<li>Use DIG High Prime DNA Labeling Kit following manufacturer's protocol</li>
<li>Denature DNA template, add labeling mix, incubate</li>
<li>Stop reaction and purify labeled probe</li>
</ul>
</li>
<li>
<p><strong>Probe Denaturation</strong></p>
</li>
<li>Heat probe at 95°C for 5 minutes</li>
<li>
<p>Chill immediately on ice for 5 minutes</p>
</li>
<li>
<p><strong>Membrane Pre-hybridization</strong></p>
</li>
<li>Place membrane in hybridization tube</li>
<li>Add pre-hybridization solution (containing blocking agent, SDS, SSC, formamide)</li>
<li>
<p>Incubate at 42°C (for formamide-containing buffer) or 65°C for 1-3 hours with rotation</p>
</li>
<li>
<p><strong>Hybridization</strong></p>
</li>
<li>Add denatured probe to fresh hybridization solution</li>
<li>Replace pre-hybridization solution with probe-containing solution</li>
<li>Hybridize overnight (12-16 hours) at appropriate temperature</li>
<li>For radioactive probes: 42°C with formamide or 65°C without</li>
<li>For DIG-labeled probes: follow manufacturer's recommendations</li>
</ol>
<h3 id="iv-washing-and-detection">IV. Washing and Detection<a class="headerlink" href="#iv-washing-and-detection" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>Post-Hybridization Washes</strong></li>
<li>Low stringency wash: 2× SSC, 0.1% SDS at room temperature, 2 × 5 minutes</li>
<li>High stringency wash: 0.1× SSC, 0.1% SDS at 65°C, 2 × 15 minutes</li>
<li>
<p>Adjust wash stringency based on probe specificity requirements</p>
</li>
<li>
<p><strong>Detection Methods</strong></p>
</li>
<li>
<p>For radioactive probes:</p>
<ul>
<li>Air-dry membrane</li>
<li>Expose to X-ray film in cassette with intensifying screens at -80°C</li>
<li>Develop film after appropriate exposure time (hours to days)</li>
<li>Alternatively, use phosphorimager for quantitative detection</li>
</ul>
</li>
<li>
<p>For DIG-labeled probes:</p>
<ul>
<li>Block membrane in blocking solution for 30 minutes</li>
<li>Incubate with anti-DIG antibody conjugated to alkaline phosphatase</li>
<li>Wash to remove unbound antibody</li>
<li>Apply chemiluminescent substrate (e.g., CDP-Star)</li>
<li>Expose to X-ray film or use imaging system</li>
</ul>
</li>
<li>
<p><strong>Signal Analysis</strong></p>
</li>
<li>Analyze band patterns and sizes relative to molecular weight markers</li>
<li>Compare with expected restriction fragment patterns</li>
<li>For quantitative analysis, use densitometry software</li>
<li>Document results with appropriate image acquisition</li>
</ol>
<h3 id="v-membrane-stripping-and-reprobing-optional">V. Membrane Stripping and Reprobing (Optional)<a class="headerlink" href="#v-membrane-stripping-and-reprobing-optional" title="Permanent link">&para;</a></h3>
<ol>
<li><strong>Stripping Procedure</strong></li>
<li>
<p>For radioactive probes:</p>
<ul>
<li>Incubate membrane in stripping solution (0.1% SDS, 0.1× SSC)</li>
<li>Heat to 95-100°C and let cool to room temperature</li>
<li>Repeat if necessary</li>
<li>Verify complete stripping by re-exposing to film</li>
</ul>
</li>
<li>
<p>For DIG-labeled probes:</p>
<ul>
<li>Wash membrane in sterile water</li>
<li>Incubate twice in 0.2M NaOH, 0.1% SDS at 37°C for 15 minutes</li>
<li>Rinse in 2× SSC</li>
</ul>
</li>
<li>
<p><strong>Reprobing</strong></p>
</li>
<li>Return to pre-hybridization step</li>
<li>Continue with new probe as described above</li>
</ol>
<p>These detailed protocols provide comprehensive step-by-step instructions for performing qPCR and Southern blot analyses, covering all aspects from sample preparation through data analysis and interpretation.</p>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "../../../../..", "features": [], "search": "../../../../../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../../../../assets/javascripts/bundle.79ae519e.min.js"></script>
      
    
  </body>
</html>