
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://CHENyiru3.github.io/Yiru_study_in_zje/MBE3_sum/Yiru/MBE%20Gene%20expression%20analysis%20technologies/">
      
      
      
      
        
      
      
      <link rel="icon" href="../../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.1">
    
    
      
        <title>MBE Gene expression analysis technologies - ZJE Collection</title>
      
    
    
      <link rel="stylesheet" href="../../../assets/stylesheets/main.484c7ddc.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#1-experimental-purpose" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../../.." title="ZJE Collection" class="md-header__button md-logo" aria-label="ZJE Collection" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            ZJE Collection
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              MBE Gene expression analysis technologies
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../../.." title="ZJE Collection" class="md-nav__button md-logo" aria-label="ZJE Collection" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    ZJE Collection
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Home
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Courses
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            
  
    Courses
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../BG2_sum/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BG2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../GP2_sum/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    GP2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../IBMS3_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IBMS3
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MBE3
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../PoN3_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    PoN3
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../IID_4_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IID_4
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../IFBS2_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IFBS2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../IN_full/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IN
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../BaO2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    BaO2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../MI2/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MI2
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Contributors
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Contributors
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../AUTHORS/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Authors
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../ZIPS_INDEX/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Zips Index
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../../CONTRIBUTING/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Contributing
  

    
  </span>
  
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#1-experimental-purpose" class="md-nav__link">
    <span class="md-ellipsis">
      
        1. Experimental Purpose
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#2-model-system" class="md-nav__link">
    <span class="md-ellipsis">
      
        2. Model System
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#3-measurement-approach" class="md-nav__link">
    <span class="md-ellipsis">
      
        3. Measurement Approach
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#4-group-setting" class="md-nav__link">
    <span class="md-ellipsis">
      
        4. Group Setting
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#5-data-analysis-presentation" class="md-nav__link">
    <span class="md-ellipsis">
      
        5. Data Analysis &amp; Presentation
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#6-technique-comparison" class="md-nav__link">
    <span class="md-ellipsis">
      
        6. Technique Comparison
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#7-complementary-usage-strategy" class="md-nav__link">
    <span class="md-ellipsis">
      
        7. Complementary Usage Strategy
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#8-technology-specific-considerations" class="md-nav__link">
    <span class="md-ellipsis">
      
        8. Technology-Specific Considerations
      
    </span>
  </a>
  
    <nav class="md-nav" aria-label="8. Technology-Specific Considerations">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#microarray" class="md-nav__link">
    <span class="md-ellipsis">
      
        Microarray
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-seq" class="md-nav__link">
    <span class="md-ellipsis">
      
        RNA-seq
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#single-cell-rna-seq" class="md-nav__link">
    <span class="md-ellipsis">
      
        Single-cell RNA-seq
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-read-sequencing" class="md-nav__link">
    <span class="md-ellipsis">
      
        Long-read Sequencing
      
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#spatial-transcriptomics" class="md-nav__link">
    <span class="md-ellipsis">
      
        Spatial Transcriptomics
      
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  



  <h1>MBE Gene expression analysis technologies</h1>

<h2 id="1-experimental-purpose">1. Experimental Purpose<a class="headerlink" href="#1-experimental-purpose" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Scientific Question</strong>: How do gene expression patterns vary across conditions, cell types, or spatial locations within tissues?</li>
<li><strong>Design Rationale</strong>: Transcriptome analysis techniques reveal comprehensive gene expression profiles, alternative splicing events, and spatial distribution of transcripts</li>
<li><strong>Follow-up Studies</strong>: Functional validation of differentially expressed genes, pathway analysis, biomarker identification, therapeutic target discovery</li>
</ul>
<h2 id="2-model-system">2. Model System<a class="headerlink" href="#2-model-system" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Primary Systems</strong>: Cell lines, tissue samples, patient biopsies, single-cell suspensions</li>
<li><strong>Rationale</strong>: These systems provide RNA of sufficient quality and quantity while maintaining biological relevance</li>
<li><strong>Alternatives</strong>:</li>
<li>In vitro cell models (pros: controlled conditions, homogeneity; cons: may not reflect in vivo complexity)</li>
<li>Animal models (pros: system-level responses; cons: species differences, ethical considerations)</li>
<li>Organoids (pros: 3D structure, cell-cell interactions; cons: incomplete tissue architecture)</li>
<li><strong>Ethical Considerations</strong>: Patient consent for clinical samples, minimizing animal use, responsible data sharing and privacy</li>
</ul>
<h2 id="3-measurement-approach">3. Measurement Approach<a class="headerlink" href="#3-measurement-approach" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Common Elements</strong>:</li>
<li>High-quality RNA extraction and preservation</li>
<li>RNA integrity verification</li>
<li>Library preparation optimization</li>
<li>Inclusion of spike-in controls</li>
<li><strong>Technical Replicates</strong>: Multiple technical replicates for microarrays; sequencing depth considerations for RNA-seq approaches</li>
<li><strong>Potential Biases</strong>:</li>
<li>RNA degradation (use RIN scores to assess quality)</li>
<li>Batch effects (include batch controls)</li>
<li>PCR amplification bias (UMIs for single-cell approaches)</li>
<li>3' bias in degraded samples (assess coverage uniformity)</li>
</ul>
<h2 id="4-group-setting">4. Group Setting<a class="headerlink" href="#4-group-setting" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Experimental Groups</strong>:</li>
<li>Test: Samples from experimental condition of interest</li>
<li>Control 1: Matched untreated/baseline samples</li>
<li>Control 2: Technical controls (spike-ins, housekeeping genes)</li>
<li>Control 3: Biological reference standards when available</li>
<li><strong>Controlled Variables</strong>: RNA quality, batch processing, sequencing platform, analysis pipeline</li>
<li><strong>Biological Replicates</strong>: Minimum 3-5 biological replicates per condition; higher numbers for heterogeneous samples</li>
<li><strong>Modified Design</strong>: Time-course analysis, dose-response relationships, multiple tissue regions</li>
</ul>
<h2 id="5-data-analysis-presentation">5. Data Analysis &amp; Presentation<a class="headerlink" href="#5-data-analysis-presentation" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Common Analysis Elements</strong>: </li>
<li>Quality control metrics (read depth, mapping rates, coverage)</li>
<li>Normalization strategies appropriate to technique</li>
<li>Differential expression analysis with statistical thresholds</li>
<li>Pathway and functional enrichment analysis</li>
<li><strong>Presentation Approaches</strong>:</li>
<li>Heatmaps for expression patterns</li>
<li>Volcano plots for significance visualization</li>
<li>PCA/t-SNE/UMAP for dimensionality reduction</li>
<li>Spatial maps for regional expression patterns</li>
</ul>
<h2 id="6-technique-comparison">6. Technique Comparison<a class="headerlink" href="#6-technique-comparison" title="Permanent link">&para;</a></h2>
<table>
<thead>
<tr>
<th>Feature</th>
<th>Microarray</th>
<th>RNA-seq</th>
<th>Single-cell RNA-seq</th>
<th>Long-read Sequencing</th>
<th>Spatial Transcriptomics</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Primary Use</strong></td>
<td>Global gene expression profiling</td>
<td>Comprehensive transcriptome analysis</td>
<td>Cell-type specific expression patterns</td>
<td>Full-length transcript analysis</td>
<td>Spatial mapping of gene expression</td>
</tr>
<tr>
<td><strong>Sensitivity</strong></td>
<td>Moderate (limited dynamic range)</td>
<td>High (wide dynamic range)</td>
<td>Moderate (limited by dropout effects)</td>
<td>Moderate (lower throughput)</td>
<td>Moderate (depends on platform)</td>
</tr>
<tr>
<td><strong>Specificity</strong></td>
<td>Good for known transcripts</td>
<td>Excellent for known and novel transcripts</td>
<td>Good for abundant transcripts</td>
<td>Excellent for isoform discrimination</td>
<td>Good for targeted gene panels</td>
</tr>
<tr>
<td><strong>Quantification</strong></td>
<td>Relative abundance through hybridization</td>
<td>Digital counts of transcript abundance</td>
<td>Digital counts at single-cell resolution</td>
<td>Accurate isoform quantification</td>
<td>Regional expression quantification</td>
</tr>
<tr>
<td><strong>Throughput</strong></td>
<td>High (thousands of genes)</td>
<td>Very high (entire transcriptome)</td>
<td>High (thousands of cells)</td>
<td>Moderate (fewer reads, longer lengths)</td>
<td>Moderate (spatial resolution trade-off)</td>
</tr>
<tr>
<td><strong>Resolution</strong></td>
<td>Gene-level only</td>
<td>Gene and transcript-level</td>
<td>Single-cell</td>
<td>Full transcript structure</td>
<td>Tissue region/cell location</td>
</tr>
<tr>
<td><strong>Cost</strong></td>
<td>Low to moderate</td>
<td>Moderate</td>
<td>High</td>
<td>High</td>
<td>Very high</td>
</tr>
<tr>
<td><strong>RNA Input Required</strong></td>
<td>50-500 ng</td>
<td>10-1000 ng</td>
<td>Single-cell (pg range)</td>
<td>50-1000 ng</td>
<td>Tissue sections</td>
</tr>
<tr>
<td><strong>Technical Expertise</strong></td>
<td>Moderate</td>
<td>Moderate to advanced</td>
<td>Advanced</td>
<td>Advanced</td>
<td>Very advanced</td>
</tr>
<tr>
<td><strong>Best For</strong></td>
<td>• Well-characterized systems<br>• Large sample comparisons<br>• Cost-effective screening<br>• Established gene sets</td>
<td>• Discovering novel transcripts<br>• Detecting rare transcripts<br>• Alternative splicing analysis<br>• Comprehensive profiling</td>
<td>• Heterogeneous samples<br>• Rare cell type identification<br>• Developmental trajectories<br>• Cellular diversity studies</td>
<td>• Isoform identification<br>• Fusion transcript detection<br>• Complex structural variants<br>• Complete transcript sequences</td>
<td>• Tissue architecture analysis<br>• Cell-cell communication<br>• Niche-specific expression<br>• Disease boundary mapping</td>
</tr>
<tr>
<td><strong>Limitations</strong></td>
<td>• Limited to known sequences<br>• Cross-hybridization issues<br>• Narrow dynamic range<br>• No novel transcript discovery</td>
<td>• Complex data analysis<br>• Computational requirements<br>• Short-read assembly challenges<br>• Batch effects</td>
<td>• Low RNA capture efficiency<br>• High dropout rates<br>• Expensive per sample<br>• Limited splicing information</td>
<td>• Lower throughput<br>• Higher error rates<br>• More expensive<br>• Specialized analysis required</td>
<td>• Limited gene coverage<br>• Resolution constraints<br>• Expensive technology<br>• Complex data integration</td>
</tr>
</tbody>
</table>
<h2 id="7-complementary-usage-strategy">7. Complementary Usage Strategy<a class="headerlink" href="#7-complementary-usage-strategy" title="Permanent link">&para;</a></h2>
<ul>
<li><strong>Initial Profiling</strong>: Use microarrays for cost-effective screening of known genes across many samples</li>
<li><strong>Comprehensive Analysis</strong>: Follow with RNA-seq for in-depth transcriptome characterization including novel transcripts</li>
<li><strong>Heterogeneity Assessment</strong>: Apply single-cell RNA-seq to dissect cellular subpopulations and rare cell types</li>
<li><strong>Structural Validation</strong>: Employ long-read sequencing to resolve complex isoforms and structural variants</li>
<li><strong>Contextual Understanding</strong>: Integrate spatial transcriptomics to map expression patterns within tissue architecture</li>
<li><strong>Integrated Approach</strong>: Design multi-platform studies for comprehensive characterization:</li>
<li>RNA-seq for global transcriptome profiling</li>
<li>Single-cell RNA-seq to resolve cellular heterogeneity</li>
<li>Long-read sequencing to characterize full-length transcripts of interest</li>
<li>Spatial transcriptomics to map key findings to tissue context</li>
</ul>
<h2 id="8-technology-specific-considerations">8. Technology-Specific Considerations<a class="headerlink" href="#8-technology-specific-considerations" title="Permanent link">&para;</a></h2>
<h3 id="microarray">Microarray<a class="headerlink" href="#microarray" title="Permanent link">&para;</a></h3>
<ul>
<li><strong>Design Optimization</strong>: Probe selection affects specificity and coverage</li>
<li><strong>Hybridization Conditions</strong>: Critical for signal-to-noise ratio</li>
<li><strong>Data Normalization</strong>: Essential for cross-array comparisons</li>
<li><strong>Legacy Data</strong>: Valuable historical datasets available for meta-analysis</li>
</ul>
<h3 id="rna-seq">RNA-seq<a class="headerlink" href="#rna-seq" title="Permanent link">&para;</a></h3>
<ul>
<li><strong>Library Preparation</strong>: Stranded vs. unstranded, poly(A) selection vs. ribo-depletion</li>
<li><strong>Sequencing Depth</strong>: Tailored to research question (15-30M reads for differential expression)</li>
<li><strong>Read Length</strong>: Longer reads improve mapping and transcript assembly</li>
<li><strong>Spike-in Controls</strong>: Essential for absolute quantification</li>
</ul>
<h3 id="single-cell-rna-seq">Single-cell RNA-seq<a class="headerlink" href="#single-cell-rna-seq" title="Permanent link">&para;</a></h3>
<ul>
<li><strong>Cell Isolation</strong>: Dissociation protocols affect cell representation</li>
<li><strong>Droplet vs. Well-based</strong>: Trade-off between cell number and coverage depth</li>
<li><strong>Doublet Rate</strong>: Quality control to remove multi-cell captures</li>
<li><strong>Computational Deconvolution</strong>: Advanced algorithms for cell type identification</li>
</ul>
<h3 id="long-read-sequencing">Long-read Sequencing<a class="headerlink" href="#long-read-sequencing" title="Permanent link">&para;</a></h3>
<ul>
<li><strong>Platform Selection</strong>: PacBio (higher accuracy) vs. Oxford Nanopore (longer reads)</li>
<li><strong>Error Correction</strong>: Critical for accurate transcript annotation</li>
<li><strong>Hybrid Approaches</strong>: Combining with short-read data for error correction</li>
<li><strong>Direct RNA Sequencing</strong>: Captures native modifications but with lower throughput</li>
</ul>
<h3 id="spatial-transcriptomics">Spatial Transcriptomics<a class="headerlink" href="#spatial-transcriptomics" title="Permanent link">&para;</a></h3>
<ul>
<li><strong>Resolution Range</strong>: From tissue regions to subcellular localization</li>
<li><strong>Coverage vs. Resolution</strong>: Trade-off between gene number and spatial detail</li>
<li><strong>Image Integration</strong>: Correlating expression with histological features</li>
<li><strong>Cell Type Deconvolution</strong>: Computational methods to resolve mixed signals</li>
</ul>
<p>This integrated framework provides a comprehensive approach to transcriptome analysis, leveraging the strengths of each technology while addressing their individual limitations. The complementary use of these methods enables researchers to build a multi-dimensional understanding of gene expression across biological systems.</p>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "../../..", "features": [], "search": "../../../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../../assets/javascripts/bundle.79ae519e.min.js"></script>
      
    
  </body>
</html>